cmd/bots/internal/common/environment.go

```diff
@@ -521,6 +532,204 @@ func executeTemplate(template string, data any, extension expectedExtension) (st
 	default:
 		return "", errors.New("unknown message type to execute a template")
 	}
+}
+
+func replaceNodeWithAlias(node string, nodesAlias map[string]string) string {
```

Comment:
Go templates should decide what to use (alias or node URL). I think it's a right way.

Reply:
See [this comment](https://github.com/wavesplatform/nodemon/pull/146#discussion_r1087756196).